_delphi_utils_python/delphi_utils/smooth.py

```diff
@@ -279,33 +303,39 @@ def left_gauss_linear_smoother(self, signal):
             signal_smoothed[signal_smoothed <= self.minval] = self.minval
         return signal_smoothed
 
-    def savgol_predict(self, signal):
+    def savgol_predict(self, signal, poly_fit_degree, nr):
         """Predict a single value using the savgol method.
-
+        
```

Comment:
this should fail linting -- we've got the utils package disabled in CI so you'll need to run it manually for now.

Reply:
Just opened #494, if that gets merged first then should probably merge it into this branch and let it run all checks.

Slight possibility when enabling these checks that an existing PR gets merged without the checks merged in and you end up with a failing main, but should be easy to resolve and won't happen once new PRs pull from the new main.

Reply:
FWIW, I now got these checks running automatically in my IDE, (thanks @chinandrew 🥇), so that will make things smoother.